main/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def board(request):
```

Comment:
the board looks not relevant to this project I would suggest: landing or welcome
```suggestion
def landing(request):
```

Reply:
Changed function name to welcome_pagw